.storybook/preview.css

```diff
@@ -86,6 +86,15 @@
   }
 }
 
+.transition-demo--accordion_content {
```

Comment:
Note: This only exists on the documentation site, it is not packaged with Optics.